File: library/src/main/java/com/example/library/dto/response/BookSyncResponse.java
package com.example.library.dto.response;
import java.time.LocalDateTime;

import jakarta.persistence.Column;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class BookSyncResponse {

    private Integer id;

    private String bookCode;
    
    private String bookName;  ////

    private Integer price;

    private Integer createdBy;
        
    private Integer categoryId;  ////
    
    private Long version;
    
    private LocalDateTime updatedAt;
    
    private LocalDateTime deleteAt;

}

File: MyApplication/app/src/main/java/com/example/myapplication/data/remote/dto/response/BookSyncResponse.kt
package com.example.myapplication.data.remote.dto.response

import java.time.LocalDateTime

data class BookSyncResponse(
    val id: Int,
    val bookCode: String,
    val bookName: String,
    val price: Int,
    val createdBy: Int,
    val version : Long,
    val categoryId: Int,
    val updatedAt: LocalDateTime,
    val deleteAt: LocalDateTime?
)